CreateTablesScript.sql

```diff
@@ -0,0 +1,118 @@
+CREATE TABLE IF NOT EXISTS `mydb`.`USERS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `email` VARCHAR(45) NOT NULL,
+  `username` VARCHAR(45) NOT NULL,
+  `password` VARCHAR(45) NOT NULL,
+  `gender` INT NOT NULL,
+  `birthdate` DATE NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE,
+  UNIQUE INDEX `password_UNIQUE` (`password` ASC) VISIBLE,
+  UNIQUE INDEX `login_UNIQUE` (`username` ASC) VISIBLE)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`PARAMETERS`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`PARAMETERS` (
```

Comment:
This table contains historical records. We need to have an automatic timestamp on each entry.